src/Microsoft.Health.Fhir.SqlServer/Features/Search/QueryCacheSetting.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Search
+{
+    internal class QueryCacheSetting
+    {
+        public const string Enabled = "enabled";
```

Comment:
nit: Maybe an enumeration

Reply:
It's true you would need something like: Enum.TryParse(x, out QueryCacheSettingsEnum cacheSetting);

But more of a suggestion.